### ISSUE TITLE
multigpu with horovod

### ISSUE THREAD

**Issue by 652994331** [NVIDIA/DeepLearningExamples#406]
Hi, recently i was using this project(pretrain) with horovod, it's good. However, i have a question, do we need more train steps in order to get the convergence when we are using horovod. it's seems we adjust the learning rate if we use horovod. for example, we have a training with lr = 5e-5(horovod) and a training with lr= 5e-5(normal single gpu), do we need more training steps for the horovod one? Thanks

**Comment by 652994331** [comment]
need more trainig steps to get same performance i mean

**Comment by swethmandava** [comment]
Yes. We make use of the **Linear Scaling Rule** rule explained [here](https://arxiv.org/pdf/1706.02677.pdf)
